### PR TITLE
Merge pull request #523 from ZecHub/JJ-Rogers-patch-78

### PR DESCRIPTION
**Pull request by frankchukwurah**
**INTRODUCTION**
<P>From a bird's-eye view, it is clear that social media has a lot more in the bag to unravel. Yes! Since its inception, it has been a force to be reckon ed with. This article is centered on "secure messengers." From Telegram to Whatsapp to imessage, these messengers are empowering connectivity across the world. Prior to the blockchain technology, the entire world was already used to web1.0 and web2.0 - Just so you know, there has been a sudden surge of interest in web3.0. In web3.0, we believe in decentralization - users will have control over their data and private keys. </p>

<h1>End-to-end encryption</h1>
<p>Alright, let's reminisce! We know how daunting sending a message was in the early days; assume you want to send a secret letter to your friend but, you don't want anyone else to read it, not even the mailman that would deliver the message. That is where end-to-end encryption comes in - Simply put, End-to-end encryption is like locking the secret letter inside a magical box that only you and your friend have a private key for. Not even the mailman can have a peek at what the letter reads.</p>

In view of the secure messengers, end-to-end encryption exists to be a security method that keeps a conversation secured. As messages travel from one device to another, no one, including the communication application you're sending the message from, and third parties won't be able to read eligible messages. Additionally, many groundbreaking and prominent messaging service providers like Facebook, Whatsapp, and Zoom are already leveraging end-to-end encryption technology. Over the past few years, some of these providers have faced controversy with regard to the decision to adopt E2EE. With end-to-end encryption technology, messaging service providers won't be able to disclose user information from their services to authorities and essentially provide private messaging to individuals involved in illicit activities.

<h2>Reasons why E2EE is necessary?</h2>

_For greater benefits, end-to-end encryption technology plays a pivotal role in situations where data security is necessary, including but not limited to finance, communication, and healthcare industries. It allows organizations to seamlessly comply with data privacy, security regulations as well as laws._
1. **Data Privacy:** With end-to-end encryption, it is said that only the intended recipient of a message or data can access and decrypt it. Meaning that even the messaging service provider, like Facebook and WhatsApp, cannot read the content.  This immersive level of data privacy is important for users and organizations that don't want sensitive information to be disclosed, it could be financial transactions, business communications, or private chat sensitive from unauthorized access.

2. **Data Security:** This strong level of technology allows for the securing of data from potential breaches even while it's still in transit or at rest.
As data travels over servers or the internet, they are exposed to many factors that make it vulnerable to interception by unauthorized access, such as hackers. With end-to-end encryption technology, the data transmitted can be intercepted but, remains unreadable as long as the sender or recipient does not hold the encryption key. This would potentially reduce the level and data breaches whilst ensuring that individual and organization communications are kept safe.

3. **Trust and User Control:** With end-to-end encryption, individuals and organizations are empowered to communicate free because they know their messages, files, and other sensitive information are kept safe from unauthorized access. Equally, the technology helps to protect individuals and organizations against government outreach and surveillance.

<h2>Some Popular Secure Messengers</h2>

* <h3>An overview of signal app</h3>
signal app
<p>Coming in hot as number one is the Signal private messenger; it is a secure messaging app with data privacy at its peak. The Signal private messenger app is free and easy to use, users do not need any prior experience. The Signal app has an end-to-end encryption technology that ensures your communication is completely private. Anyone looking to send private messages securely should consider using the Signal private messenger app because it  will exponentially reduce both your cybersecurity vulnerabilities and your data footprint.</p>

* <h3>An overview of Bison Relay app</h3>
BisonRelay
<p>The Bison Relay app made its debut in 2022. Bison Relay is not dependent on users having accounts - Insanely unbelievable! Here, every message that is sent and received is individually paid for over the Decred Lightning Network. The sole purpose of this app is to completely eliminate the need for accounts whilst minimizing metadata. On the other hand, Bison Relay put a stop to the way censorship is typically implemented. Message content. </p>

* <h3>An overview of session app</h3>
Session App
<p> If you don't already know what Session app is, don't fret! Here is a glimpse, Session is one of the vying end-to-end encrypted messengers. The app was built to minimize sensitive metadata. The session app was designed for individuals who want absolute privacy, and freedom from any form of surveillance. On the Session app, users can create an account without phone numbers or email addresses which means it is completely anonymous. Users can send messages safely via their network without any trace so, there's nothing to leak.</p>

<h2>Wrapping Up: Secure Messengers</h2>

In summary, end-to-end encryption is important because it fosters robust privacy protection, reinforces data security to its core, and enables users to hand full control over their own information as they travel over networks and devices, thereby addressing critical concerns in the digital age.

### CODE REVIEW

**Comment by JJ-Rogers** [comment]
added to privacy tools section: https://github.com/ZecHub/zechub/blob/main/site/guides/privacytools/secure%20messengers.md
